### PR TITLE
fix IN operator to use OR instead of AND

### LINKED ISSUE #10: Operators: The in operator is supposed to work as an OR not an AND
**Describe the bug**
when using the in operator it generates a query like so `X=3 AND Y=4`

**To Reproduce**
Use the in operator with multiple values

**Expected behavior**
the query should be `X=3 OR Y=4`

### PR DESCRIPTION
**Pull request by realiarthur**
Resolves #10

### CODE REVIEW

File: src/operators/in.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ class InOperator extends Operator {
 
   public exec() {
     // In operator resembles and and exactly
-    const andOperator = new AndOperator(this.value);
-    return andOperator.exec();
+    const orOperator = new OrOperator(this.value);
+    return orOperator.exec();
```
Comment (khaledosama999) [inline src/operators/in.ts:14]: Update the comment on line 12 please, so it says it resembles `an or operator`

**Review by khaledosama999 — CHANGES_REQUESTED** [review]

**Comment by khaledosama999** [comment]
@realiarthur Overall great job I just have one minor request instead of using `git commit` to commit please run `npm run git-cz` and add a fix commit, I will merge it on dev and then create a release

**Comment by realiarthur** [comment]
@khaledosama999 I have updated everything that you said. Thank you!

**Comment by khaledosama999** [comment]
@realiarthur  Approved, will make a new release with your fix

**Review by khaledosama999 — APPROVED** [review]

**Comment by khaledosama999** [comment]
Github actions is down, release will be up when it's back

**Pull request merged**